src/com/itacademy/eshop/Runner.java

```diff
@@ -9,7 +11,7 @@
 
 public class Runner {
 
-    public void run() throws ProductNotFoundException {
+    public void run() throws ProductNotFoundException, DuplicateProductException {
```

Comment:
this method already handles DuplicateProductException, so it does not need to throw it